src/renderers/webgl/WebGLUtils.js

```diff
@@ -1,140 +1,212 @@
 /**
  * @author thespite / http://www.twitter.com/thespite
+ * @author fernandojsg / http://fernandojsg.com
  */
+ /* jshint esversion: 6 */
+
+import * as Constants from '../../constants';
+import { WebGLConstants } from './WebGLConstants';
+
+//------------------------------------------------------------------------------
+// THREE TO WEBGL
+//------------------------------------------------------------------------------
+var THREE_TO_WEBGL = {
+
+	[ Constants.NearestFilter ]: WebGLConstants.NEAREST,
```

Comment:
Was not aware of this `[ variable ]` thing. Is it well supported?

Reply:
It's ES6 computed literals. I thought gulp would do the transform itself but it doesn't seems so :\

Reply:
The ugly but compatible version is to put the value of the constant as is, or try to use a gulp pluging to convert it to its literal value when creating the bundle. @mrdoob thoughts?

Reply:
My fault, this is es2015 and it's supported in all the browsers, but on firefox we need to add extra parenthesis: `[ (value) ]`. I'll fix it

Reply:
actually, on the latest version of nightly it works as expected :)

Reply:
@mrdoob my fault, I was testing it incorrectly on firefox O:) it's working as expected in all the browsers, also tested on mobile (android & ios).